osu.Game/Screens/Select/BeatmapClearScoresDialog.cs

```diff
@@ -39,6 +39,28 @@ public BeatmapClearScoresDialog(BeatmapInfo beatmap, Action onCompletion)
             };
         }
 
+        public BeatmapClearScoresDialog(ScoreInfo score, Action onCompletion)
```

Comment:
I'm a bit opposed against this landing in here. This new dialog is arguably *not* a `BeatmapClearScoresDialog` since it doesn't really clear scores from a beatmap. I'd argue this class needs at least a rename, if not some proper genericizing and refactoring.

Reply:
OK. Will fix it. I was also debating whether I want to create a new class for it or not.

How does `ClearLocalScoreDialog` sound for this?

Reply:
Sure, I'm fine with that name.

Reply:
Clear generally refers to clearing more than one object. Delete is better IMO.

Reply:
Yeah, I realized that a bit after. I have changed it to `LocalScoreDeleteDialog`.